KsApi/mutations/PostCommentMutation.swift

```diff
@@ -0,0 +1,22 @@
+import Foundation
+
+public struct PostCommentMutation<T: GraphMutationInput>: GraphMutation {
```

Comment:
Noticed we aren't testing any of these GraphMutation types. Is that intentional?

Reply:
Added some basic assertions on the properties.